src/Bicep.Core.UnitTests/Analyzers/Linter/ApiVersions/AzureResourceApiVersionTests.cs

```diff
@@ -18,7 +18,6 @@ public class AzureResourceApiVersionTests
         [DataRow("9999-12-31-preview", "9999-12-31", "-preview")]
         [DataRow("9999-12-31-privatepreview", "9999-12-31", "-privatepreview")]
         [DataRow("9999-12-31-beta", "9999-12-31", "-beta")]
-        [DataRow("9999-12-31-alpha", "9999-12-31", "-alpha")]
```

Comment:
>DataRow [](http://example.com/codeflow?start=9&length=7)

We really liked testing some of these rows twice, didn't we?

Reply:
Sometimes you've just got to be REALLY sure things are working